docs/index.md

```diff
@@ -39,32 +39,21 @@ to you with the software itself.
 
 - **Manufacturer Name**: {{device_manufacturer}}
 - **Manufacturer's Address**: 17 Grampian Place Tatton NSW 2650 Australia
-- **Software as a Medical Device Identification**: {{device_name}} {{device_version}}
+- **Software as a Medical Device Identification**: {{device_name}}
+  {{device_version}}
 
 ### Difference to the open source version
 
-This device corresponds to the software downloadable directly from
-<https://radiotherapy.ai/> (Regulated Medical Device). It does **not**
-correspond to the software downloadable from
-<https://github.com/RadiotherapyAI/rai> (Open Source Unregulated
-Software).
+This device corresponds to the {{device_name}} software that is downloadable
+directly from <{{device_website}}>. It does **not** correspond to the
+{{open_source_software_name}} software that is downloadable from
+<{{online_repository}}>.
 
-The Open Source Unregulated Software is provided in the hope that it might be
-useful but WITHOUT ANY WARRANTY. It is **not** intended for clinical use, and
-is instead intended for research use only.
+The {{open_source_software_name}} is provided in the hope that
+it might be useful but WITHOUT ANY WARRANTY. It is **not** intended for
+clinical use, and is instead intended for research use only.
 
-The Regulated Medical Device described by this documentation corresponds only
-to the product that is able to be downloaded from <https://radiotherapy.ai/>.
-This is the product that is intended to be able to be used for clinical use
-according to the intended purpose described within this documentation.
-
-### Workflow overview
-
-Radiotherapy AI's {{device_name}} product (Regulated Medical Device) is
```

Comment:
The vendor should be a configuration item

Reply:
Yup see above. This comment is on the removed text, not the updated text.